pkg/routes/logs.go

```diff
@@ -42,10 +44,27 @@ func (l Logs) Install(c *restful.Container) {
 func logFileHandler(req *restful.Request, resp *restful.Response) {
 	logdir := "/var/log"
 	actual := path.Join(logdir, req.PathParameter("logpath"))
+
+	// pre check file, if file name is too long, return 404
+	isNameOversize := preCheckLogFileNameLength(actual)
+	if isNameOversize {
```

Comment:
Minor naming nit: Rename function so it makes sense when used inline, e.g.?

`if logFileNameIsTooLong(actual) {` ?

Reply:
yeah, too redundant here, thanks!